### PR TITLE
docs(helm): Fix docs spelling error

### PR DESCRIPTION
**Pull request by jiaweizhou**

### CODE REVIEW

**Review by jascott1 — APPROVED** [review]
LGTM Thanks @jiaweizhou

**Comment by jiaweizhou** [comment]
Thanks for your review @jascott1

**Review by bacongobbler — APPROVED** [review]
LGTM as well. I (and @jascott1) are new Helm maintainers so I'll defer to someone else to merge for the time being until I'm up to speed. Thanks!

**Review by michelleN — APPROVED** [review]

**Pull request merged**
